src/server/frontend_xwayland/xcb_connection.h

```diff
@@ -89,6 +90,7 @@ class XCBConnection
         std::atomic<xcb_atom_t> mutable atom{XCB_ATOM_NONE};
     };
 
+    /// Takes ownership of the given FD
```

Comment:
I know this is pre-existing, but we _ought_ to have a way to document this in code. (A "unique" variant of the "shared" `mir::Fd`, say.)

Reply:
I believe `mir::Fd` is sufficient. The only reason we need to document ownership is so we know who should close it. I believe in another branch I'm already refactoring this to use a Mir FD, which you'll see at some point.

Reply:
Maybe shared ownership it is sufficient here. But since C++ acquired proper move semantics representing unique, transferable ownership is also meaningful.